include/pmix/pmix_common.h.in

```diff
@@ -183,7 +183,7 @@ BEGIN_C_DECLS
 
 /****    PMIX ERROR CONSTANTS    ****/
 /* PMIx errors are always negative, with 0 reserved for success */
-#define PMIX_ERROR_MIN  -41  // set equal to number of non-zero entries in enum
+#define PMIX_ERROR_MIN  -42  // set equal to number of non-zero entries in enum
```

Comment:
We are a C99 package - those are C99 comments. You'll find them throughout the code.